dist/index.js

```diff
@@ -4896,7 +4899,7 @@ async function getElixirVersions() {
     .split('\n')
     .forEach((line) => {
       const elixirMatch =
-        line.match(/^(.+)-otp-([^ ]+)/) || line.match(/^([^ ]+)/)
+        line.match(/^v?(.+)-otp-([^ ]+)/) || line.match(/^v?([^ ]+)/)
```

Comment:
This is where we remove the `v` from the listing (this way we allow `v1.12` as well as `1.12`).